jenkins/Jenkinsfile

```diff
@@ -150,7 +150,7 @@ pipeline {
             description: "The path for the bucket where the custom Dataproc image generation logs will be stored")
         string(name: "GCE_IMAGE_BUCKET", defaultValue: "gs://leo-gce-image-creation-logs",
             description: "The path for the bucket where the custom GCE image generation logs will be stored")
-        string(name: "DATAPROC_VERSIONS", defaultValue: "1.2.79-debian9 1.4.51-debian10",
+        string(name: "DATAPROC_VERSIONS", defaultValue: "1.4.51-debian10",
```

Comment:
Since there will be a single Dataproc version going forward, the two loopings with

```groovy
for (String dp_version : DATAPROC_VERSIONS.trim().split(' ')) {
```

should be removed.

Reply:
I'd prefer to keep this becuz we still potentially will need it if we were to upgrade dataproc version (for a while, we'll need to support 2 versions again)